package.json

```diff
@@ -39,17 +39,20 @@
   },
   "homepage": "https://github.com/filecoin-project/filecoin-station#readme",
   "dependencies": {
-    "@glif/filecoin-address": "^2.0.1",
+    "@glif/filecoin-address": "^2.0.20",
```

Comment:
Is it necessary to bump `@glif/filecoin-address` from `2.0.1` to `2.0.20`? I think this is the root cause of the Buffer problem, isn't it? Maybe we should solve this upgrade first in https://github.com/filecoin-station/filecoin-station/pull/240, in isolation from other changes in this PR. What do you think?

Reply:
Downgrading this module back to `2.0.1` seems to fix the problem. The GUI starts fine, and `npm run build` works too.

I noticed one warning in the DevTools console, I think it's not a problem, but it could be a hint on how to fix the Buffer issue for the new Filecoin-address version.

<img width="1022" alt="Screenshot 2023-01-05 at 14 37 43" src="https://user-images.githubusercontent.com/1140553/210792795-1747c0bd-0211-410a-93c1-cb02823af2c3.png">

Reply:
See https://github.com/filecoin-station/filecoin-station/pull/379

Reply:
the downgrade worked 🙌 

Also the `bufer.Buffer` warning seems not to create any issues 👍